packages/plugins/Scale/README.md

```diff
@@ -9,3 +9,8 @@ The scale plugin shows the current map resolution as relative scale (1:x) and/or
 ## Configuration
 
 For details on the `displayComponent` attribute, refer to the [Global Plugin Parameters](../../core/README.md#global-plugin-parameters) section of `@polar/core`.
```

Comment:
In other plugins this is written below the table. Mind moving it there as well? :)

Reply:
https://github.com/Dataport/polar/pull/144/commits/bc9722477d4861bf26bbcffd160703a529e286cc